udata/templates/macros/issues.html

```diff
@@ -1,8 +1,8 @@
 {% macro issues_btn(obj, icons=None) %}
-{% set basename = obj.__class__.__name__|lower %}
 <button type="button" class="btn btn-danger btn-issues"
     data-api-url="{{ url_for('api.issues', for=obj.id|string) }}"
-    data-subject="{{ obj.id }}"
+    data-subject-id="{{ obj.id }}"
+    data-subject-class="{{ obj.__class__.__name__ }}"
```

Comment:
Yes, we can, but I find `__class__.__name__` explicit and consistent with the fact that we are avoiding necessary inheritance (one of the purpose of this PR).
I think a property hide the fact that it's simply the raw class name.
It's debatable, what do you think about it ?